octopusdeploy_framework/schemas/step_template.go

```diff
@@ -142,15 +143,18 @@ func GetStepTemplateParameterResourceSchema() rs.ListNestedAttribute {
 		Required:    true,
 		NestedObject: rs.NestedAttributeObject{
 			Attributes: map[string]rs.Attribute{
-				"default_value": rs.StringAttribute{
-					Description: "A default value for the parameter, if applicable. This can be a hard-coded value or a variable reference.",
-					Optional:    true,
-					Computed:    true,
-					Default:     stringdefault.StaticString(""),
-					PlanModifiers: []planmodifier.String{
-						stringplanmodifier.UseStateForUnknown(),
-					},
-				},
+				"default_value": util.ResourceString().
+					Description("A default value for the parameter, if applicable. This can be a hard-coded value or a variable reference.").
+					Optional().
+					Computed().
+					Default(stringdefault.StaticString("")).
+					PlanModifiers(stringplanmodifier.UseStateForUnknown()).
+					Build(),
+				"default_sensitive_value": util.ResourceString().
```

Comment:
These schemas always confuse me, should the default_sensitive_value also use the "PlanModifiers(stringplanmodifier.UseStateForUnknown())"?